src/plugins/analysis/information_leaks/code/information_leaks.py

```diff
@@ -1,60 +1,58 @@
+from __future__ import annotations
+
 import re
 from itertools import chain
 
 from analysis.PluginBase import AnalysisBasePlugin
+from helperFunctions.virtual_file_path import get_top_of_virtual_path
 from objects.file import FileObject
 
 PATH_REGEX = {
     'user_paths': re.compile(rb'/home/[^%\n \x00]+'),
     'root_path': re.compile(rb'/root/[^%\n \x00]+'),
-    'www_path': re.compile(rb'/var/www/[^\n \x00]+')
+    'www_path': re.compile(rb'/var/www/[^\n \x00]+'),
 }
 
-FILES_REGEX = {
-    'any_history': re.compile(rb'.+_history')
-}
+FILES_REGEX = {'any_history': re.compile(rb'.+_history')}
```

Comment:
nit: Formatting. For consistency with the others this should not be a single line.